src/setupTests.js

```diff
@@ -3,20 +3,17 @@ import React from "react";
 import { configure, mount } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import { Provider } from "react-redux";
-import { ConnectedRouter } from "connected-react-router";
-
-import configureStoreDev, { history } from "./store/configureStore";
+import { MemoryRouter } from "react-router";
+import configureStore, { configureStoreTest } from "./store/configureStore";
```

Comment:
why is `configureStoreTest` imported here??

Reply:
I have removed the unused import